tests/pos/i1665.scala

```diff
@@ -0,0 +1,12 @@
+
+import scala.reflect.runtime.universe._
+import scala.reflect.runtime.{currentMirror => cm}
+
+object Test extends App {
+  val mutant = new { val x = 2 }
+  val c = cm.classSymbol(mutant.getClass)
```

Comment:
@nicolasstucki - could you update this test to use Java reflection instead? Cheers from Dima